bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteEditorCssReference.java

```diff
@@ -0,0 +1,27 @@
+package de.agilecoders.wicket.extensions.markup.html.bootstrap.editor;
+
+import org.apache.wicket.request.resource.CssResourceReference;
+
+public class SummernoteEditorCssReference extends CssResourceReference {
```

Comment:
DropZoneFileUpload also stores the library locally and I think as long as the implementation doesn't change we can load it from the applications server.